### DISCUSSION TITLE
Notepad in a visible place

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#5944] [Ideas]
I'm working with the buildin notepad and it's hidden in small plus sign on the right side.
I'm using also the notepad in the device's overview but it will also help if the general Notepad from the rightside will be in a more visible place.
Thanks.

**Comment by si458** [comment]
What builtin notepad?
Do u mean the notes button?
Any screenshots?

**Reply by ghost** [reply]
The notes thingy on the right side where all the templates are.
![Screenshot_2](https://github.com/Ylianst/MeshCentral/assets/102313998/45238a08-bc5a-478f-a705-6c6589c3f4b4)

I'm using it a lot and it would nice if it can be in a more visible place,

**Reply by si458** [reply]
oh wow! learn something new every day haha!
its called 'Personal Notes' not notepad (thats what confused me)

you can already display device notes by using `showNotesPanel: true` under `domains` of your `config.json`, this will display notes on the device overview page (but you cant edit them without clicking the notes button)

but sadly your `personal notes` are a different thing, how else would you like to access them quicker?
